xen-posix/flags/minios-cflags.tmp.in

```diff
@@ -0,0 +1 @@
+-D__XEN_INTERFACE_VERSION__=0x00030205 -D__INSIDE_MINIOS__ -I%{prefix}%/include/mirage-xen-posix/include
```

Comment:
where did this come from? Seems possibly duplicated

Reply:
The `-I%{prefix}%/include/mirage-xen-posix/include` come from the `mirage-xen-posix` dependency in `mirage-xen-minios.pc`.